src/main/resources/swagger.api/ecs-tlr.yaml

```diff
@@ -63,6 +92,17 @@ components:
             $ref: 'schemas/EcsTlr.yaml#/EcsTlr'
     badRequestResponse:
       description: Validation errors
+      content:
+        application/json:
+          example:
+            errors:
+              - message: Request is invalid
+                code: notfound.request
```

Comment:
This probably fits better in 404. And message in 404 probably should not say "Request is invalid"